### DISCUSSION TITLE
a small typo

### DISCUSSION THREAD

**Discussion by rohit-ganapathy** [Jeiwan/uniswapv3-book#21] [Milestone 0. Introduction]
just wanted to highlight a small typo in the section "Milestone 0. Introduction":

"There are two roles, liquidity providers and trades,...."

i'm wondering if it should be 

"There are two roles, liquidity providers and **traders**,...."

**Comment by Jeiwan** [comment]
Thanks!
It was fixed in this PR: https://github.com/Jeiwan/uniswapv3-book/pull/18
